test/C/src/NativeListsAndTimes.lf

```diff
@@ -2,22 +2,25 @@ target C
 
 // This test passes if it is successfully compiled into valid target code.
 main reactor(
-    x: int(0),
-    y: time(0),                // Units are missing but not required
-    z(1 msec),                 // Type is missing but not required
-    p: int[](1, 2, 3, 4),      // List of integers
-    q: interval_t[](1 msec, 2 msec, 3 msec),  // list of time values
-    g: time[](1 msec, 2 msec)  // List of time values
+    x: int = 0,
+    y: time = 0,                   // Units are missing but not required
+    z = 1 msec,                    // Type is missing but not required
+    p: int[] = {1, 2, 3, 4},       // List of integers
+    q: interval_t[] = {1 msec, 2 msec, 3 msec},  // list of time values
+    g: time[] = {1 msec, 2 msec},  // List of time values
+    g2: int[] = {}
```

Comment:
```suggestion
```
See above comment; this is C23 syntax that we should not rely on

Reply:
Thanks for your input on this, I had no idea about this limitation of C.

Do you think the LF validator should forbid this/ emit a warning, or just ignore it and leave it up to the C compiler to decide whether that's valid?

Reply:
I think the latter. We should not try to do the job of the target compiler.